src/test/compile-fail/kindck-impl-type-params.rs

```diff
@@ -27,12 +27,14 @@ fn f<T>(val: T) {
     let t: S<T> = S(marker::PhantomData);
     let a = &t as &Gettable<T>;
     //~^ ERROR : std::marker::Send` is not satisfied
+    //~^^ ERROR : std::marker::Copy` is not satisfied
```

Comment:
FYI it can be nice to use the "arrow form" here for successive lines,  ie `//~| ...` instead of `//~^^ ...`